### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by dehowef**
Updated README with more examples and useful information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,38 +23,132 @@ var config = {
...
+Users can also use named prepared statements. Using prepared statements allows query execution plans to be cached on the server on a per connection statement. Once a query name is sent to the server, it will be parsed and executed. On subsequent sends the query will skip the parse request and instead send the name of the query:
+```js
+...
+const query = {
+  name: 'fetch-data',
+  text: 'MATCH(n) WHERE name = $1 RETURN n',
+  values: ['Eve']
+}
+
+client.query(fetch-data, (err,res) => {
```
Comment (bylee5) [inline README.md]: Is fetch-data possible?

Comment (dehowef) [reply README.md]: Oops, I made an error, which I corrected. Thanks for pointing that out

File: README.md [hunk 2]
```diff
@@ -20,38 +23,131 @@ var config = {
...
+To create data in AgensGraph, the query() API function is again used to query the database. Below is an example of creating Cypher data by passing text directly to client.query():
+```js
+var ag = require('agensgraph');
+const client = new ag.Client(Config);
 
-client.query('DROP GRAPH IF EXISTS gpt CASCADE');
+client.connect();
 client.query('CREATE GRAPH gpt');
 client.query('SET graph_path = gpt');
+client.query("CREATE (a:person {name:'Alice')-[r:knows {from: 'school'}]->(b:person {name: 'Bob'} RETURN a,r,b)", function (err, res) {
```
Comment (bylee5) [inline README.md]: CREATE statement appears to have a syntax error.

Comment (dehowef) [reply README.md]: Addressed this error

File: README.md [hunk 3]
```diff
@@ -20,34 +23,127 @@ var config = {
...
+## Creating Data
+To create data in AgensGraph, the query() API function is again used to query the database. Below is an example of creating Cypher data by passing text directly to client.query():
+```js
+var ag = require('agensgraph');
+const client = new ag.Client(Config);
+
+client.connect();
 client.query('CREATE GRAPH gpt');
 client.query('SET graph_path = gpt');
+client.query("CREATE (a:person {name:'Alice')-[r:knows {from: 'school'}]->(b:person {name: 'Bob'}) RETURN a,r,b", function (err, res) {
```
Comment (bylee5) [inline README.md]: CREATE statement appears to have a syntax error.

**Review by bylee5 — APPROVED** [review]

**Pull request merged**